tsconfig.json

```diff
@@ -21,6 +21,8 @@
       "indico/modules/events/editing/*": ["./indico/modules/events/editing/client/js/*"],
       "indico/modules/events/management/*": ["./indico/modules/events/management/client/js/*"],
       "indico/modules/events/persons/*": ["./indico/modules/events/persons/client/js/*"],
+      "indico/modules/events/sessions/*": ["./indico/modules/events/sessions/client/js/*"],
+      "indico/modules/events/timetable/*": ["./indico/modules/events/timetable/client/js/*"],
```

Comment:
Please also add this to the import resolver map in `.eslintrc.yml` (at some point we should get a CI job to ensure all these places are in sync)

Reply:
yeah there are at least 3 places now with these mappings (eslint, jsconfig & tsconfig)